agent/agentssh/agentssh.go

```diff
@@ -80,15 +82,23 @@ func NewServer(ctx context.Context, logger slog.Logger, maxTimeout time.Duration
 	if err != nil {
 		return nil, err
 	}
+	if x11SocketDir == "" {
+		x11SocketDir = filepath.Join(os.TempDir(), ".X11-unix")
+	}
+	err = fs.MkdirAll(x11SocketDir, 0o700)
```

Comment:
The dir shouldn't be made until an X11 forwarding requests comes through since most people won't use X11 forwarding.

Reply:
Agreed